Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -577,7 +577,10 @@ var ListView = React.createClass({
     }
     if (updatedFrames) {
       updatedFrames.forEach((newFrame) => {
-        this._childFrames[newFrame.index] = merge(newFrame);
+        // Sometimes x/y/width/heigh maybe return zero
+        if (newFrame.x != 0 || newFrame.y != 0 || newFrame.width != 0 || newFrame.height != 0) {
```

Comment:
Looks like this would exclude frames that have one of their coordinate as 0. That doesn't sound right.
Could you check only for zero size?